subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -796,10 +796,48 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
         /* Those are handled by NTVDM */
         case SCS_WOW_BINARY:
         {
-            DisplayMessage(L"Trying to load '%S'.\n"
-                           L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
-                           L"Press 'OK' to continue.",
-                           ProgramName);
+            static const PCSTR AppName = "\"%ProgramFiles%\\otvdm\\otvdmw.exe\" ";
+
+            STARTUPINFOA si;
+            PROCESS_INFORMATION pi;
+            CHAR ExpName[MAX_PATH];
+
+            ExpandEnvironmentStringsA(AppName, ExpName, ARRAYSIZE(ExpName) - 1);
+            strcat(ExpName, "\"");         // Add double-quote before ProgramName
+            strcat(ExpName, ProgramName);  // Append Program name
+            strcat(ExpName, "\"");         // Add double-quote after ProgramName
+
+            ZeroMemory(&pi, sizeof(pi));
+            ZeroMemory(&si, sizeof(si));
+            si.cb = sizeof(si);
+
+            /* Create the process */
+            if (CreateProcessA(NULL,    // No Application Name
+                               ExpName, // Just our Command Line
+                               NULL,    // Cannot inherit Process Handle
+                               NULL,    // Cannot inherit Thread Handle
+                               FALSE,   // No handle inheritance
+                               0,       // No extra creation flags
+                               NULL,    // No environment block
+                               NULL,    // No starting directory 
+                               &si,
+                               &pi))
+            {
+                /* Close the handles */
+                CloseHandle(pi.hThread);
+                CloseHandle(pi.hProcess);
+                break;
+            }
+            else
+            {
+                /* Retrieve the actual path to the "Program Files" directory for displaying the error */
+                ExpandEnvironmentStringsA("%ProgramFiles%", ExpName, ARRAYSIZE(ExpName) - 1);
+
+                DisplayMessage(L"Trying to load '%S'.\n"
+                               L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
+                               L"Consider installing OTVDM in '%S'. Press 'OK' to continue.",
```

Comment:
I'd suggest to name it "winevdm" because actually it's how it called.

https://github.com/otya128/winevdm

Reply:
But people have more focused on this other one:
https://github.com/otya128/otvdm
🤔 🤔

Reply:
https://reactos.org/wiki/User:Petr-akhlamov/WOW

Reply:
OK then what should @peterooch do? He packaged an OTVDM (or winevdm?) setup, see https://github.com/reactos/rapps-db/pull/159 , but it seems it was mainly named OTVDM.

Reply:
![image](https://user-images.githubusercontent.com/578406/142871738-ec87f7e6-ad23-4e72-a7ab-264d605903cf.png)

Reply:
The developer didn't renamed binaries produced in winevdm repo (they're still named otvdm), and this leads to confusion.

Reply:
Btw, the developer also has an old repo named "otvdm", which is a deprecated version of otvdm: https://github.com/otya128/OTVDM. So if to think logically, the name for the current one appears to be WineVDM. :slightly_smiling_face:

Reply:
I edited the error message as @HBelusca requested and changed one line to the following:
"Consider installing WineVDM from the ReactOS Applications Manager in\n'%S'.\n\n"